# HG changeset patch
# User Pantxo Diribarne <pantxo.diribarne@gmail.com>
# Date 1425407354 -3600
#      Tue Mar 03 19:29:14 2015 +0100
# Node ID 90299974feda87dabb86499a0bab4a8c45ebee51
# Parent  ac18e88ea1bf5efdeb591c4d917d32d13419c8e0
Change default uipanel "position" for compatibility with Matlab (bug #44411)

* graphics.cc (default_panel_position): change to default position to [0 0 1 1]

diff --git a/libinterp/corefcn/graphics.cc b/libinterp/corefcn/graphics.cc
--- a/libinterp/corefcn/graphics.cc
+++ b/libinterp/corefcn/graphics.cc
@@ -448,18 +448,18 @@ default_control_sliderstep (void)
 
 static Matrix
 default_panel_position (void)
 {
   Matrix retval (1, 4, 0.0);
 
   retval(0) = 0;
   retval(1) = 0;
-  retval(2) = 0.5;
-  retval(3) = 0.5;
+  retval(2) = 1;
+  retval(3) = 1;
 
   return retval;
 }
 
 static double
 convert_font_size (double font_size, const caseless_str& from_units,
                    const caseless_str& to_units, double parent_height = 0)
 {
